flask/main.py

```diff
@@ -150,6 +151,10 @@ def organization():
 def connect():
     return "flask /connect"
 
+@app.route('/product/0/info', methods=['GET'])
+def product_info():
```

Comment:
@bfpepper87 does this provide a response? I don't see one, but maybe I'm missing something. This is erroring at scale https://testorg-az.sentry.io/issues/3966708105/events/5aee332b1cd441b0aa76effc1807f9d7/?project=5808655 can you fix?